File: packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/WebviewFlutterPlugin.java
package io.flutter.plugins.webviewflutter;

import io.flutter.plugin.common.PluginRegistry.Registrar;

/** WebviewFlutterPlugin */
public class WebviewFlutterPlugin {
  /** Plugin registration. */
  public static void registerWith(Registrar registrar) {
    registrar
        .platformViewRegistry()
        .registerViewFactory(
            "plugins.flutter.io/webview", new WebViewFactory(registrar.messenger()));
  }
}

File: packages/webview_flutter/ios/Classes/WebViewFlutterPlugin.m
#import "WebViewFlutterPlugin.h"
#import "FlutterWebView.h"
#import "FLTCookieManager.h"

@implementation FLTWebViewFlutterPlugin

+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  FLTWebViewFactory* webviewFactory =
      [[FLTWebViewFactory alloc] initWithMessenger:registrar.messenger];
  [registrar registerViewFactory:webviewFactory withId:@"plugins.flutter.io/webview"];
  FLTCookieManager* cookieManager = [[FLTCookieManager alloc] init:registrar.messenger];
}

@end
